fuzzers/005-tilegrid/generate_full.py

```diff
@@ -499,13 +504,12 @@ def add_tile_bits(tile_db, baseaddr, offset, frames, words, height=None):
 
     bits = tile_db['bits']
     block_type = util.addr2btype(baseaddr)
-
     assert 0 <= offset <= 100, offset
     assert 1 <= words <= 101
     assert offset + words <= 101, (
         tile_db, offset + words, offset, words, block_type)
 
-    assert block_type not in bits
+    #assert block_type not in bits:
```

Comment:
We should handle more explicitly

Reply:
Take a look at my IOB PR. I had some logic there for doing a merge check on these. We should maybe also move this utility function to somewhere shared since its in generate_full.py and add_tdb.py. Or maybe just import from one of the existing .py files instead of duplicating

Reply:
Done